docs/modules/ROOT/pages/includes/quarkus-wiremock.adoc

```diff
@@ -86,5 +86,19 @@ endif::add-copy-button-to-env-var[]
 --|string
 |`src/test/resources`
 
+a|icon:lock[title=Fixed at build time] [[quarkus-wiremock_quarkus.wiremock.devservices.global-response-templating]]`link:#quarkus-wiremock_quarkus.wiremock.devservices.global-response-templating[quarkus.wiremock.devservices.global-response-templating]`
 
-|===
\ No newline at end of file
+[.description]
+--
+If global response templating should be enabled for WireMock, see https://wiremock.org/2.x/docs/response-templating/[https://wiremock.org/2.x/docs/response-templating/]
```

Comment:
Maybe we can add a hint, that `Response templating` is enabled by default, but not globally and link the [3.X doc](https://wiremock.org/3.x/docs/response-templating/)

Reply:
Obsolete